Backend/app/rider-platform/rider-app/Main/spec/Storage/MultiModal.yaml

```
@@ -183,3 +190,85 @@ WalkLegMultimodal:
+    distance: (Kernel.Types.Common.Distance distance distanceUnit)|E
+    fromStopDetails: (Kernel.External.MultiModal.Interface.Types.MultiModalStopDetails <$> fromStopCode <*> fromStopGtfsId <*> gtfsId)|E
+    toStopDetails: (Kernel.External.MultiModal.Interface.Types.MultiModalStopDetails <$> toStopCode <*> toStopName <*> toStopGtfsId)|E
+    routeDetails: (Kernel.External.MultiModal.Interface.Types.MultiModalRouteDetails <$> routeGtfsId <*> routeLongName <*> routeShortName)|E
+    agency: (Kernel.External.MultiModal.Interface.Types.MultiModalAgency <$> agencyGtfsId <*> agencyName)|E)
+    startLocation: Kernel.External.Maps.Google.MapsClient.LatLngV2 <$> startLocationLat <*> startLocationLon|E
+    endLocation: Kernel.External.Maps.Google.MapsClient.LatLngV2 <$> endLocationLat <*> endLocationLon|E
+
+  toTType:
+    distance: ((.value))|I
+    distanceUnit: ((.unit))|I
+    fromStopCode:  (fromStopDetails >>= (.stopCode))|E
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)